Controller/Redirect/In.php

```diff
@@ -197,7 +197,6 @@ public function execute()
             $this->sdkProxy->paymentCapture($lastTransactionId, $rvvupPaymentId);
         } catch (\Exception $e) {
             $this->orderManagement->cancel($orderId);
```

Comment:
I think we don't do this, incase the rvvup order did capture but there was some other issue where it error internally after capture and we got a error resposne (or a network blip). Then Rvvup would treat it as success send a webhook but the order is cancelled in magento, causing us to fail the update.

Leave it as is, and in this edge case/issues, it just solves it self when the payment is expired etc.